eventsource/stores/sqlstore/store.go

```diff
@@ -14,10 +14,11 @@ type store struct {
 }
 
 const (
-	saveSQL = "INSERT INTO %s (aggregate_id, sequence_id, created_at, user_id, type, data) VALUES ($1, $2, $3, $4, $5, $6)"
-	loadAggregateSQL = "SELECT aggregate_id, sequence_id, created_at, user_id, type, data FROM %s WHERE aggregate_id = $1 ORDER BY sequence_id ASC LIMIT 100000"
-	loadBySequenceIDSQL = "SELECT aggregate_id, sequence_id, created_at, user_id, type, data FROM %s WHERE sequence_id > $1 ORDER BY sequence_id ASC LIMIT 100000"
-	loadByTimestampSQL = "SELECT aggregate_id, sequence_id, created_at, user_id, type, data FROM %s WHERE timestamp > $1 ORDER BY timestamp ASC LIMIT 100000"
+	saveSQL                    = "INSERT INTO %s (aggregate_id, sequence_id, created_at, user_id, type, data) VALUES ($1, $2, $3, $4, $5, $6)"
+	loadAggregateSQL           = "SELECT aggregate_id, sequence_id, created_at, user_id, type, data FROM %s WHERE aggregate_id = $1 ORDER BY sequence_id ASC LIMIT 100000"
+	loadBySequenceIDSQL        = "SELECT aggregate_id, sequence_id, created_at, user_id, type, data FROM %s WHERE sequence_id > $1 ORDER BY sequence_id ASC LIMIT 1000"
```

Comment:
Why the limitation?

Reply:
See https://github.com/SKF/go-eventsource/pull/31/files/2fb075345283b77c2523648dc6fcfbf83cab0929#r250258077